paper/paper.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "pylhe: A Lightweight Python interface to Les Houches Event files"
+tags:
+  - Python
+  - physics
+  - high energy physics
+authors:
+  - name: Alexander Puck Neuwirth^[Corresponding author.]
+    orcid: 0000-0002-2484-1328
+    affiliation: "1, 2"
+  - name: Matthew Feickert
+    orcid: 0000-0003-4124-7862
+    affiliation: 3
+  - name: Lukas Heinrich
+    orcid: 0000-0002-4048-7584
+    affiliation: 4
```

Comment:
I also requested a review by @lukasheinrich since he is listed as author too.

Reply:
In any case I reckon you can proceed with 3 good-to-go :).